pkg/convert/arb_test.go

```diff
@@ -175,14 +186,15 @@ func Test_FromArb(t *testing.T) {
         }
       }`),
 			expectedErr: fmt.Errorf("unsupported value type 'map[string]interface {}' for key '@@locale'"),
+			original:    false,
 		},
 	}
 	for _, tt := range tests {
 		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
 
-			actual, err := FromArb(tt.input)
+			actual, err := FromArb(tt.input, tt.original)
```

Comment:
Keep it simple! No need to declare `original` variable.

```suggestion
			actual, err := FromArb(tt.input, false)
```

Reply:
Oki, I will change then in another tests too  ;)